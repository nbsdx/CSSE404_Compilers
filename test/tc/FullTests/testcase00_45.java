class foo {
	public static void main(String[] args) {
		System.out.println(bar.fib(4));
	}
}

class baz {
	public int fib(int a) {
		return a;
	}
}

class bar extends baz {
	public boolean fib2(int a) {
		return baz.fib(a);
	}
}