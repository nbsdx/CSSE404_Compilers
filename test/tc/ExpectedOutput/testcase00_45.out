Variable bar is not declared.
No method named fib found for class null
In MiniJava, System.out.println only takes an int.  The expression has type null
Variable baz is not declared.
No method named fib found for class null
Actual return type null of method fib2 does not match declared type boolean
